src/main/java/org/allenai/wikitables/WikiTablesDataProcessor.java

```diff
@@ -66,18 +102,84 @@
     return entityLinking;
   }
 
+  static void addDerivations(List<CustomExample> dataset, int beamSize) {
+    // Parses the examples in the given dataset, and stores all the correct derivations.
+
+    // Setting all the options typically selected by Sempre
```

Comment:
It sucks that we have to set all of these options. Is there not an easier way to do this?